Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.cpp

```diff
@@ -537,6 +537,9 @@ RefPtr<ServiceWorkerFetchTask> NetworkConnectionToWebProcess::createFetchTask(Ne
 
 void NetworkConnectionToWebProcess::scheduleResourceLoad(NetworkResourceLoadParameters&& loadParameters, std::optional<NetworkResourceLoadIdentifier> existingLoaderToResume)
 {
+    if (UNLIKELY(!m_networkProcess->allowsFirstPartyForCookies(m_webProcessIdentifier, loadParameters.request.firstPartyForCookies())))
+        RELEASE_LOG_ERROR_IF(storageSession() && storageSession()->trackingPreventionDebugLoggingEnabled(), Loading, "scheduleResourceLoad: Web process does not have cookie access to url %" PRIVATE_LOG_STRING " for request %" PRIVATE_LOG_STRING, loadParameters.request.firstPartyForCookies().string().utf8().data(), loadParameters.request.url().string().utf8().data());
```

Comment:
I am not convinced it is OK to log URL / domains, even if only for internal devices.

Maybe using SENSITIVE_LOG_STRING and asking the users that see this to enable sensitive logging. But private logging doesn't seem right to me.